packages/product-feed-persoo/tests/Unit/PersooFeedItemTest.php

```diff
@@ -0,0 +1,258 @@
+    }
+
+    public function testMinimalPersooFeedItemIsCreatable()
+    {
+        $persooFeedItem = $this->persooFeedItemFactory->create($this->defaultProduct, $this->defaultDomain);
+
+        self::assertInstanceOf(PersooFeedItem::class, $persooFeedItem);
+        self::assertEquals(self::PRODUCT_ID, $persooFeedItem->getId());
+        self::assertEquals(self::PRODUCT_ID, $persooFeedItem->getSeekId());
+        self::assertEquals(self::PRODUCT_NAME, $persooFeedItem->getTitle());
+        self::assertEquals(self::PRODUCT_URL, $persooFeedItem->getLink());
+        self::assertNull($persooFeedItem->getDescription());
+        self::assertEquals([0 => self::MAIN_CATEGORY_ID], $persooFeedItem->getHierarchyIdsIndexedByCategoryId());
+        self::assertEquals([0 => self::MAIN_CATEGORY_NAME], $persooFeedItem->getHierarchyNamesIndexedByCategoryId());
+        self::assertNull($persooFeedItem->getImageLink());
+        self::assertEquals(t('In stock'), $persooFeedItem->getAvailability());
+        self::assertThat($persooFeedItem->getPrice()->getPriceWithoutVat(), new IsMoneyEqual(Money::zero()));
+        self::assertThat($persooFeedItem->getPrice()->getPriceWithVat(), new IsMoneyEqual(Money::zero()));
+        self::assertEquals(self::EUR, $persooFeedItem->getCurrency()->getCode());
+        self::assertNull($persooFeedItem->getBrand());
+        self::assertEquals(['Flag_name'], $persooFeedItem->getFlagNames());
+        self::assertEquals([self::PARAMETER_NAME => self::PARAMETER_VALUE], $persooFeedItem->getProductParameterValuesIndexedByName());
+    }
+
+    public function testPersooFeedItemWithBrand()
+    {
+        /** @var \Shopsys\FrameworkBundle\Model\Product\Brand\Brand|\PHPUnit\Framework\MockObject\MockObject $brand */
+        $brand = $this->createMock(Brand::class);
+        $brand->method('getName')->willReturn(self::BRAND_NAME);
+        $this->defaultProduct->method('getBrand')->willReturn($brand);
+
+        $persooFeedItem = $this->persooFeedItemFactory->create($this->defaultProduct, $this->defaultDomain);
+
+        self::assertEquals(self::BRAND_NAME, $persooFeedItem->getBrand());
+    }
+
+    public function testPersooFeedItemWithDescription()
+    {
+        $this->defaultProduct->method('getDescription')
+            ->with(self::PRODUCT_ID)->willReturn(self::PRODUCT_DESCRIPTION);
+
+        $persooFeedItem = $this->persooFeedItemFactory->create($this->defaultProduct, $this->defaultDomain);
+
+        self::assertEquals(self::PRODUCT_DESCRIPTION, $persooFeedItem->getDescription());
+    }
+
+    public function testPersooFeedItemWithImageLink()
+    {
+        $this->mockProductImageUrl($this->defaultProduct, $this->defaultDomain, self::IMAGE_URL);
+
+        $persooFeedItem = $this->persooFeedItemFactory->create($this->defaultProduct, $this->defaultDomain);
+
+        self::assertEquals(self::IMAGE_URL, $persooFeedItem->getImageLink());
+    }
+
+    public function testPersooFeedItemWithSellingDenied()
```

Comment:
I believe this test deserves a data provider for both cases ('Out of stock' and 'In stock')

Reply:
The first test - the minimal one tests the other version, so it is not necessary